doc/start/os-recommendations.rst

```diff
@@ -27,16 +27,20 @@ Linux Kernel
 
   Older kernel client versions may not support your `CRUSH tunables`_ profile
   or other newer features of the Ceph cluster, requiring the storage cluster to
-  be configured with those features disabled. For RBD, a kernel of version 5.3
+  be configured with those features disabled. For RBD, a kernel of version 5.4
```

Comment:
What is the reason for this unrelated change?  I don't mind it because 5.4 is an LTS kernel and 5.3 isn't and therefore long out of support, but I still wanted to ask.